internal/setup/setup.go

```diff
@@ -664,7 +741,7 @@ func Select(release *Release, slices []SliceKey) (*Selection, error) {
 		for newPath, newInfo := range new.Contents {
 			if old, ok := paths[newPath]; ok {
 				oldInfo := old.Contents[newPath]
-				if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath) && new.Package != old.Package {
+				if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath) && new.Package != old.Package {
```

Comment:
Need a test to see the path infos clashing if one is generate and another is not.

Reply:
Added in 8334b5d.

Reply:
But we are still checking the glob distance, right?

Reply:
So we are checking glob distance, but that's in the `validate()` function:

```go
func (r *Release) validate() error {
...
	for _, pkg := range r.Packages {
		for _, new := range pkg.Slices {
			keys = append(keys, SliceKey{pkg.Name, new.Name})
			for newPath, newInfo := range new.Contents {
				if old, ok := paths[newPath]; ok {
					oldInfo := old.Contents[newPath]
					if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath) && new.Package != old.Package {
						if old.Package > new.Package || old.Package == new.Package && old.Name > new.Name {
							old, new = new, old
						}
						return fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
					}
				} else {
					if newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath {
						globs[newPath] = new
					}
					paths[newPath] = new
				}
			}
		}
	}
...
```